specification/compute/resource-manager/Microsoft.Compute/stable/2021-03-01/compute.json

```diff
@@ -10112,6 +10125,10 @@
         "modelAsString": true
       }
     },
+    "spotRestorePolicy": {
```

Comment:
You need to include a new [Example ](https://github.com/Azure/azure-rest-api-specs/tree/compute-2021-03/specification/compute/resource-manager/Microsoft.Compute/stable/2021-03-01/examples)file for this new property.